packages/contracts/contracts/TransactionManager.sol

```diff
@@ -370,7 +371,7 @@ contract TransactionManager is ReentrancyGuard, ITransactionManager {
       }
 
       // Handle receiver chain external calls if needed
-      if (txData.callDataHash == keccak256(new bytes(0))) {
+      if (txData.callTo == address(0)) {
```

Comment:
We should change the name of `callTo` to be like `fulfillHelperAddress` or something more descriptive.